AES.md

```
@@ -0,0 +1,199 @@
+# Plan: Developing a C++ Static Library for Detecting Potential File Encryption
+
+## 1. Objective
+
+The primary objective is to develop C++ libraries:
+  - static (`.a`)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)